### PR TITLE
Added Yelp and tested it

### PR DESCRIPTION
**Pull request by Maryom**

### CODE REVIEW

File: Appz/Appz/Apps/Yelp.swift [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                    pathComponents: ["app"],
+                    queryParameters: [:]
+                ),
+                web: Path()
+            )
+            
+        case .Search(let query):
+            return ActionPaths(
+                app: Path(
+                    pathComponents: ["", "search"],
```
Comment (Mazyod) [inline Appz/Appz/Apps/Yelp.swift:54]: @Maryom I think there is an extra empty string here, right? Is it really needed?

File: Appz/Appz/Apps/Yelp.swift [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                web: Path(
+                    pathComponents: ["search"],
+                    queryParameters: ["terms": query]
+                )
+            )
+            
+        case .SearchLocation(let query, let loc):
+            return ActionPaths(
+                app: Path(
+                    pathComponents: ["", "search"],
```
Comment (Mazyod) [inline Appz/Appz/Apps/Yelp.swift:66]: @Maryom Same thing goes to the empty string here

Comment (Maryom) [inline Appz/Appz/Apps/Yelp.swift:66]: @Mazyod Good morning, I checked this URL: http://www.yelp.com/biz/yelp-san-francisco taken from https://www.yelp.com/developers/documentation/v2/phone_search then I tried it with and without empty String and it gave me same result so I didn't add it. So do you think I need to add it?

Comment (Mazyod) [inline Appz/Appz/Apps/Yelp.swift:66]: @Maryom I don't understand, what do you need to add? I just think the empty string in the arrays should probably be removed.

**Pull request merged**